src/main/kotlin/model/Cars.kt

```diff
@@ -1,21 +1,14 @@
 package model
 
-import generator.RandomGenerator
+class Cars(cars: List<Car>) {
+    private val _cars: List<Car> = cars
+    val cars: List<Car> get() = _cars
```

Comment:
```suggestion
class Cars(val cars: List<Car>) {
```
Cars 클래스에는 add를 하는 로직이 없습니다.
그렇다면 굳이 backing property 구조로 작성할 필요가 없어 보이네요.